openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js

```diff
@@ -786,7 +786,7 @@ describe('Glossary page should work properly', () => {
       .should('contain', term3);
 
     cy.get('[data-testid="governance"]').click();
-    cy.get('[data-testid="appbar-item-glossary"]').click();
+    cy.get('[data-testid="app-bar-item-glossary"]').click({ force: true });
```

Comment:
Why we are using force: true heavily? 
We can use cy.clickOnOutside to clear popover if any

Reply:
Outside click is also closing or the menu list that, the reason we are using this.